client/components/plans/plan-actions/index.jsx

```diff
@@ -172,9 +173,17 @@ module.exports = React.createClass( {
 		return (
 			<div>
 				<button className="button is-primary plan-actions__upgrade-button"
-					onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'button' ) }>
-						{ this.translate( 'Upgrade Now', { context: 'Store action' } ) }
+					onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: true } ), 'button' ) }>
+						{ this.translate( 'Start Free Trial', { context: 'Store action' } ) }
 				</button>
+
+				<small className="plan-actions__trial-period">
+					{ this.translate( 'Try it free for 14 days, no credit card needed, or', { context: 'Store action' } ) }&nbsp;
+					<a href='#'
+						onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'link' ) }>
+						{ this.translate( 'upgrade now', { context: 'Store action' } ) }
```

Comment:
Like drew said somewhere in this PR, it should be okay without capitalization since it's a part of the sentence above.